README.md

```diff
@@ -87,6 +87,21 @@ f5_vip 'myvip' do
   #                   anything else is a named firewall_policy
   staged_firewall_policy
 end
+
+# Create a F5 monitor template
+f5_monitor 'test-monitor' do 
+  # Required properties
+  template_type 'TTYPE_HTTP' # F5 template type
+  parent_template 'http'
+  interval 5 # frequency to check in seconds
+  timeout 3 # seconds to respond
+  
+  #optional properties (default values are shown)
+  dest_ip '0.0.0.0' # tells f5 what address to test. Default indicates to test each node in pool 
+  dest_port '0' # tells f5 which port to test; Default indicates to test all ports that are open
```

Comment:
Same note as above, I think this is the same as `*` in the UI which means to test the port this is attached to. So if you have a node `ip:80` in the pool that this is on, it'll test 80.